crates/sui-framework/sources/crypto.move

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::crypto {
+    /// verify signature, if valid, recovered return public key, otherwise throw error.
+    public native fun ecrecover(signature: vector<u8>, hashed_msg: vector<u8>): vector<u8>;
```

Comment:
@joyqvq I'd suggest splitting each new Move functionality (algorithm) in different PRs